app/styles/components/_spinner.scss

```diff
@@ -0,0 +1,54 @@
+$duration: 0.5s;
+$spinner-color: $wds-color-link;
+$z-side-nav-overlay: $z-8;
+
+.wds-spinner {
+	@keyframes rotator {
+		from {
+			transform: rotate(0);
+		}
+		to {
+			transform: rotate(360deg);
+		}
+	}
+
+	@keyframes dash {
+		to {
+			stroke-dashoffset: 0;
+		}
+	}
+
+	animation: rotator $duration linear infinite;
+	left: 50%;
+	margin: -33px 0 0 -33px;
```

Comment:
The size is set by JS code (a couple of things have to be calculated for it to work).

This is pretty old CSS, I've given it a quick try and couldn't fix it without the margin. We've got one very big spinner and a couple of smaller ones and they seem to be working with this margin.

I'll note it and if I don't get back to it I'll create a ticket.